cli/webpack.config.js

```diff
@@ -0,0 +1,38 @@
+var path = require('path');
```

Comment:
webpack supports non-js config files if you have the proper compiler installed in node_modules. This could be a `.ls` file